### DISCUSSION TITLE
Preventing channel change by a commissioner on a Thread network

### DISCUSSION THREAD

**Discussion by suveshpratapa** [openthread/openthread#7661] [Ideas]
I wanted to start a discussion on whether it is possible for us to prevent a channel change on a Thread network.  Say a Thread device is on two networks at once, on the same channel.  If a Thread commissioner induces a channel change on the Thread network, then this could cause network disruption and/or other problems.

OpenThread currently has a channel manager feature, however, per my understanding, turning this feature off still does not prevent a Thread commissioner from changing the pending dataset or channel on the network.

We could prototype some application code that detects a pending dataset channel change, and revert it if needed, or even prevent the channel change from happening on the radio, however, this would likely break Thread 1.1 certification, which assumes a commissioner can always change a Thread channel.

Is it possible to explore a Thread spec addition to include a security policy bit that prevents a channel change in such use cases?

**Comment by suveshpratapa** [comment]
cc @mnp222 @romacdon @jwhui

**Comment by jwhui** [comment]
You are correct that OpenThread's Channel Manager feature does not prevent channel changes. The Channel Manager feature actually does the opposite and looks for opportunities to change the channel based on measured channel conditions across the set of available channels. Whether or not to disallow channel changes is a good discussion to have.

That said, I think discussions around potential Thread Specification updates are better done within the Thread Group.

**Comment by suveshpratapa** [comment]
Thank you @jwhui ! I think I'll open a SPEC JIRA item for discussion.
